.github/labeler.yml

```diff
@@ -0,0 +1,63 @@
+"Area: boards":
+  - boards/**/*
+
+"Area: build system":
+  - Makefile.*
+  - makefiles/**/*
+
+"Area: CI":
+  - .circleci/**/*
+  - .github/**/*
```

Comment:
`.github/workflow/*.yml`. There are other metadata (e.g. issue templates) in `.github`.